zettarepl/utils/logging.py

```diff
@@ -12,7 +12,7 @@ class LongStringsFilter(logging.Filter):
     def __init__(self, name=""):
         super().__init__(name)
 
-        self.max_string_length = int(os.environ.get("LOGGING_MAX_STRING_LENGTH", "64"))
+        self.max_string_length = int(os.environ.get("LOGGING_MAX_STRING_LENGTH", "512"))
```

Comment:
Q: wouldn't it be better to have something like this be a system tunable (assuming those exist for `zettarepl`) instead of having to change a hard coded value in the src every time we bump up against the high water mark ?

Reply:
I would like to have stuff like this in tunables, but William is against this and he has a point, we might get bogged down in investigating tickets with weird and unsupported tunable values.

This value specifically is unlikely to change in the future, it must only cover ZFS CLI errors which are usually very short (but not as ridiculously short as 64 bytes, I probably wanted log lines to fit one line of 120 characters width terminal or something when I first wrote it)

Reply:
Noted: thx for the excellent explanation, @themylogin 😁